client/src/components/AccountPrompt/index.js

```diff
@@ -16,15 +16,16 @@ import { login, useAuth } from '../../auth'
 
 const AccountPrompt = ({ text, apiEndpoint, pageHeader, linkText, link }) => {
   const [isLoading, setIsLoading] = useState(false)
-  const [username, setUsername] = useState('')
+  const [user, setUser] = useState('')
   const [password, setPassword] = useState('')
   const { apiUrl } = useConfiguration()
   const history = useHistory()
   const [logged] = useAuth()
 
-  const isFormInvalid = !username || !password
+  const isFormInvalid = !user || !password
 
   async function onSubmit(e) {
+    const username = user.trim()
```

Comment:
you could have also kept `username`, and then called this variable something like `trimmedUsername`